portfolio/src/main/webapp/index.html

```diff
@@ -11,7 +11,7 @@
     <script src="script.js"></script>
   </head>
 
-  <body onload=initBody() style="background-image: url('/images/background.jpg');">
+  <body onload=initBody(); style="background-image: url('/images/background.jpg');">
```

Comment:
Is the semicolon required?

Reply:
no its just for style when having multiple statements in a tag